apps/studio/components/interfaces/App/FeaturePreview/FeaturePreviewModal.tsx

```diff
@@ -34,15 +41,33 @@ const FeaturePreviewModal = () => {
         : TelemetryActions.FEATURE_PREVIEW_ENABLED,
       properties: { feature: selectedFeatureKey },
     })
+
+    if (selectedFeatureKey === LOCAL_STORAGE_KEYS.UI_SQL_EDITOR_TABS) {
+      removeTabsByEditor(ref as string | undefined, 'table')
+    }
+    if (selectedFeatureKey === LOCAL_STORAGE_KEYS.UI_SQL_EDITOR_TABS) {
```

Comment:
One of these `ifs` should be about `sql`?

Reply:
?

Reply:
ah yes, thanks!